src/pages/projects/forms/ProjectDetailsForm.tsx

```diff
@@ -179,6 +181,34 @@ const ProjectDetailsForm: FC<Props> = ({ formik, project, isEdit }) => {
               ),
             }}
           />
+          <div>
+            <NetworkSelectorForm
+              value={formik.values.default_project_network}
+              setValue={(value) =>
+                void formik.setFieldValue("default_project_network", value)
+              }
+              selectProps={{
+                label: "Default network",
+                disabled:
+                  (formik.values.features_profiles === false &&
+                    features === "customised") ||
+                  isEdit,
+                help: isEdit ? (
+                  <>
+                    Edit the network in the{" "}
+                    <ResourceLink
+                      type="profile"
+                      value="default"
+                      to={`/ui/project/${project?.name}/profile/default`}
+                    />{" "}
+                    {" profile"}
+                  </>
+                ) : (
+                  ""
+                ),
+              }}
+            />
+          </div>
```

Comment:
The extra div wrapper should not be necessary.
```suggestion
            <NetworkSelectorForm
              value={formik.values.default_project_network}
              setValue={(value) =>
                void formik.setFieldValue("default_project_network", value)
              }
              selectProps={{
                label: "Default network",
                disabled:
                  (formik.values.features_profiles === false &&
                    features === "customised") ||
                  isEdit,
                help: isEdit ? (
                  <>
                    Edit the network in the{" "}
                    <ResourceLink
                      type="profile"
                      value="default"
                      to={`/ui/project/${project?.name}/profile/default`}
                    />{" "}
                    {" profile"}
                  </>
                ) : (
                  ""
                ),
              }}
            />
```